### ISSUE TITLE
[Setting - Startup]: Screen reader is not announcing the sub-text information for 'Launch size' and 'Launch parameter'.

### ISSUE THREAD

**Issue by Priyanka-Chauhan123** [microsoft/terminal#15981]
### Windows Terminal version

1.19.2481.0

### Windows build number

23529.1000

### Other Software

**Test Environment:**
OS: Windows 11 Version Dev (OS Build 23529.1000)
App: Windows Terminal Preview

### Steps to reproduce

**Repro Steps:**

1.Open Windows Terminal.
2.Open Settings page using 'Ctr+,'
3. Navigate to the Startup page.
5. Navigate over the 'Launch size' and 'Launch parameter' using Narrator and observe the issue.

**User Experience: **
When a screen reader does not provide complete information, it is difficult for people using assistive technology (like screen readers) to understand the functionality of the control. Having complete information about the control will help users to use the control as per their required task.

**MAS Reference:**
https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships

**Attachment :**
[Screen reader is not announcing the sub-text information for 'Launch size' and 'Launch parameter'..zip](https://github.com/microsoft/terminal/files/12644573/Screen.reader.is.not.announcing.the.sub-text.information.for.Launch.size.and.Launch.parameter.zip)

### Expected Behavior

Screen reader should announce the sub-text information for 'Launch size' and 'Launch parameter'.

### Actual Behavior

Screen reader is not announcing the sub-text information for 'Launch size' and 'Launch parameter'.

**Comment by Priyanka-Chauhan123** [comment]
Logged by mistake
